docs/modules/plugins/partials/plugin-web-app-steps.adoc

```diff
@@ -84,6 +84,41 @@ Given I am on main application page
 When I go to relative URL `/products/new`
 ----
 
+=== Web page steps
+
+==== Refresh page
+
+Refreshes the page. Browser reloads current page which is the same action as one caused by pressing F5 on the keyboard.
+
+[source,gherkin]
+----
+When I refresh page
+----
+
+==== Verify page title
+
+Checks that the page's *title* matches to *text* according to the provided string validation rule
```

Comment:
also we don't use bold in the step descriptions

Reply:
removed